docs/toc.md

```diff
@@ -261,7 +261,7 @@
 ### [🔧 Tuples and unnamed types](csharp/tuples.md)
 ### [🔧 Interfaces](csharp/interfaces.md)
 ### [Methods and Lambda Expressions](csharp/methods-lambda-expressions.md)
-#### [🔧 methods](csharp/methods.md)
+#### [🔧 Methods](csharp/methods.md)
```

Comment:
@mairaw You can remove the wrench -- this has the C# 7.0 version of the topic, which includes tuples.